src/components/CanvasSaveButton/CanvasSaveButton.js

```diff
@@ -363,26 +375,45 @@ class CanvasSaveButton extends PureComponent {
           );
         });
 
-        // North arrow
-        let p = Promise.resolve();
-        if (destContext && extraData && extraData.northArrow) {
-          p = this.drawNorthArrow(destContext, destCanvas);
+        // Custom info
+        let logoPromise = Promise.resolve();
+        if (destContext && extraData && extraData.logo) {
+          logoPromise = this.drawElement(
+            extraData.logo,
+            destContext,
+            destCanvas,
+          );
         }
 
-        p.then((arrowWidth) => {
-          // Copyright
-          if (
-            destContext &&
-            extraData &&
-            extraData.copyright &&
-            extraData.copyright.text
-          ) {
-            const maxWidth = arrowWidth
-              ? destContext.canvas.width - arrowWidth
-              : destContext.canvas.width;
-            this.drawCopyright(destContext, destCanvas, maxWidth);
+        logoPromise.then((customItemSize = [0, 0]) => {
+          // North arrow
+          let arrowPromise = Promise.resolve();
+          if (destContext && extraData && extraData.northArrow) {
+            arrowPromise = this.drawElement(
+              extraData.northArrow,
+              destContext,
```

Comment:
the context comes from the canvas  so you don't have to pass it as parameter. Get it directly from the canvas